src/remote_adapter/remote_adapter_client/CMakeLists.txt

```diff
@@ -1,15 +1,52 @@
 project(RemoteTestingAdapterClient)
 cmake_minimum_required(VERSION 3.11)
+include(FetchContent)
+
+# Enable PIC as static libraries may be linked to shared objects
+set (CMAKE_POSITION_INDEPENDENT_CODE TRUE)
 set (CMAKE_CXX_STANDARD 11)
+set (CMAKE_INCLUDE_CURRENT_DIR ON)
+set (CMAKE_AUTOMOC ON)
+
+FetchContent_Declare(
+  rpclib
+  GIT_REPOSITORY https://github.com/rpclib/rpclib.git
+  GIT_TAG        v2.2.1
+)
+
+FetchContent_GetProperties(rpclib)
+if(NOT rpclib_POPULATED)
+  FetchContent_Populate(rpclib)
+  add_subdirectory("${rpclib_SOURCE_DIR}" "${rpclib_BINARY_DIR}")
+  find_package( Threads REQUIRED )
+  add_library(rpc::rpclib ALIAS rpc)
+endif()
 
+find_package(Qt5 COMPONENTS Core REQUIRED)
 find_package( Threads REQUIRED )
 FIND_PACKAGE(Lua 5.2 EXACT REQUIRED)
-message(STATUS "Lua found : " ${LUA_INCLUDE_DIR})
+
+# list(GET LUA_LIBRARIES 0 LUA_LIB)
```

Comment:
@aderiabin please remove this commented part( lines 29..42) since lua imported library already provided in root cmake list

Reply:
@EKuliiev Current part of code was moved into if condition, because it is needed for separate compilation of module `remote_adapter_client`
(see 497ec85ac829fefb39ccf5198d0ad6ac798e7b74)